app/src/lib/feature-flag.ts

```diff
@@ -73,6 +73,11 @@ export function enableResetToCommit(): boolean {
   return enableDevelopmentFeatures()
 }
 
+/** Should we allow checking out a single commit? */
+export function enableCheckoutCommit(): boolean {
+  return enableDevelopmentFeatures()
```

Comment:
```suggestion
  return enableBetaFeatures()
```

Go ahead and set for next beta